packages/sonarwhal/src/lib/config.ts

```diff
@@ -265,11 +267,19 @@ export class SonarwhalConfig {
         const ignoredUrls = loadIgnoredUrls(userConfig);
         const rules = normalizeRules(userConfig.rules);
 
-        validateRules(rules, userConfig);
-
         return new SonarwhalConfig(userConfig, browsers, ignoredUrls, normalizeRules(rules));
     }
 
+    /**
+     * Return a list of rules that have invalid configurations.
+     * @param config
+     */
+    public static validateRuleConfig(config: UserConfig): Array<string> {
```

Comment:
Shouldn't this be `validateRulesConfig`?

Reply:
Also, do we need to normalize here? I thought this was done earlier in the process.